Enable All Streams (#12)
scryptedapp/arlo: README.md

## General Setup Notes

* Ensure that your Arlo account's default 2FA option is set to either SMS or email.

> * Motion events notifications must be turned on in the Arlo app. If you are receiving motion push notifications, Scrypted will also receive motion events. If you are using any downstream plugins (e.g. Homekit), you can disable the Arlo app notifications at the operating system level after they've been enabled inside of the Arlo app. This way, you are not getting double notifications for motion events.

Reviewer: ```suggestion
* Motion events notifications must be turned on in the Arlo app. If you are receiving motion push notifications, Scrypted will also receive motion events. If you are using any downstream plugins (e.g. Homekit), you can disable the Arlo app notifications in your phone's notification settings after they've been enabled inside of the Arlo app. This way, you are not getting double notifications for motion events.
```

* Disable smart detection and any cloud/local recording in the Arlo app. Arlo Cloud permits one active RTSP/DASH stream, so any smart detection or recording features may prevent downstream plugins (e.g. Homekit) from successfully pulling the video feed after a motion event.
* It is highly recommended to enable the Rebroadcast plugin to allow multiple downstream plugins (e.g. Homekit) to pull the video feed within Scrypted.

> * The plugin supports pulling WebRTC, RTSP, or DASH streams from Arlo Cloud. It is recommended to use WebRTC for streaming and RTSP for recording. DASH is inconsistent in reliability, and may return finicky codecs that require additional FFmpeg output arguments, e.g. `-vcodec h264`. *Note that both RTSP and DASH will ultimately pull the same video stream feed from your camera, and they cannot both be used at the same time due to the single stream limitation. If you have the stream open while Scrypted tries to record, it will not work. WebRTC does not meet this requirement so seperate streams of WebRTC and RTSP/DASH are possible for recording and streaming.*

Reviewer: Is it true that you can stream in both WebRTC and RTSP concurrently? Just thinking about when scrypted consumes these streams, not including the cloud recording

Author: > Is it true that you can stream in both WebRTC and RTSP concurrently? Just thinking about when scrypted consumes these streams, not including the cloud recording

It is not, I just tried again. I guess I was doing something wrong when I was testing before. It returns an error that the stream is not available because a SIP call is in progress. I will update everything accordingly that if the stream is open period, recording will not work and that you cannot have two streams to the same camera open at one time.

Reviewer: Got it, thanks for checking. Yeah I think it's fine to add this note in the README, no need to handle it in the code

Reviewer: nah I don't think we need to handle this further, just printing the arlo error should be sufficient
